sql/src/main/antlr/OpenDistroSQLParser.g4

```diff
@@ -116,6 +117,11 @@ orderByElement
     : expression order=(ASC | DESC)? (NULLS (FIRST | LAST))?
     ;
 
+limitClause
+    : LIMIT (offset=decimalLiteral COMMA)? limit=decimalLiteral
+    | LIMIT limit=decimalLiteral (OFFSET offset=decimalLiteral)?
```

Comment:
In this case, OFFSET clause should be required? Otherwise these 2 cases may have overlap: https://github.com/antlr/grammars-v4/blob/master/sql/mysql/Positive-Technologies/MySqlParser.g4#L1083

Reply:
Yep they have one case overlapped `LIMIT limit=decimal`. The benefit is offset token (optional) and limit token both exist, so we don't have to separately overwrite two methods for these two cases to build AST.